src/common/state/table.cc

```diff
@@ -81,8 +81,13 @@ int64_t table_timeout_handler(event_loop *loop,
 
   CHECK(callback_data->retry.num_retries >= 0 ||
         callback_data->retry.num_retries == -1);
-  LOG_WARN("retrying operation %s, retry_count = %d", callback_data->label,
-           callback_data->retry.num_retries);
+  if (callback_data->retry.timeout > HEARTBEAT_TIMEOUT_MILLISECONDS) {
```

Comment:
Yes. If you have any ideas on how to do this better, I'm open. Maybe comparing the label string?